internal/compiler/parser.rs

```diff
@@ -880,6 +880,11 @@ impl Spanned for Option<SyntaxToken> {
     }
 }
 
+/// return true if experimental parser feature should be activated
+fn enable_experimental() -> bool {
+    std::env::var_os("SLINT_EXPERIMENTAL_SYNTAX").map_or(false, |x| !x.is_empty())
```

Comment:
I hope that this is a fast function in the standard library and doesn’t involve parsing the entire environment over and over again. Otherwise it might be worthwhile to cache the result, since this function is now called from what might be hot paths in the parser?

Reply:
This shouldn't be in a hot path. It is only called when using the new syntax anyway, so shouldn't be called in "production" at  all.

Reply:
Ah yes! I see now:)